nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -25,6 +26,22 @@ def check_processing_module_name(processing_module: ProcessingModule):
         )
 
 
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=NWBFile)
+def check_session_start_time(nwbfile: NWBFile):
+    """Check if the session_start_time was set to an appropriate value."""
+    if nwbfile.session_start_time <= datetime(1980, 1, 1).astimezone():
```

Comment:
Is Jan. 1, 1980 an OK threshold for determining a potentially invalid neurophysiology experiment? Or should we go back further?

Reply:
I think that's fine.